internal/backend/runtime/omni/controllers/omni/cluster_workload_proxy_status_test.go

```diff
@@ -218,11 +227,50 @@ func (m *mockWorkloadProxyReconciler) Reconcile(cluster resource.ID, aliasToUpst
 		m.data = map[resource.ID]map[string][]string{}
 	}
 
-	m.data[cluster] = aliasToUpstreamAddresses
+	newCluster := true
+
+	for als := range rd.AliasPort {
+		for clusterID, alsToHost := range m.data {
+			if _, ok := alsToHost[als]; ok && clusterID != cluster {
+				panic(fmt.Errorf("alias %q already exists and used by cluster %q", als, clusterID))
+			}
+
+			if clusterID == cluster {
+				m.t.Logf("replacing data for cluster %q :: %v :: %v", cluster, rd.Hosts, rd.AliasPort)
```

Comment:
do we need all these logs for every change every time reconcile is called (it will be called excessively)

Reply:
I guess it's fine for the test code 🤔

Reply:
This is test package, so it doesn't mater that much?